shared/message/platform/webots/messages.proto

```diff
@@ -83,20 +85,30 @@ message Message {
     string      text         = 2;
 }
 
+/// NUbots specific data sent from our own controllers rather than the official RoboCup Webots controller
+message OdometryGroundTruth {
+    /// Indicates if this message exists
+    bool exists = 1;
+    /// Affine3d transform from odometry (world) space to the robot (torso) space.
+    /// It measures the world in robot space
+    mat4 Htw = 2;
+}
+
 message SensorMeasurements {
     // simulation time stamp at which the measurements were performed expressed in [ms] from the start of the connection
     uint32 time = 1;
     // real unix time stamp at which the measurements were performed in [ms]
-    uint64                             real_time        = 2;
-    repeated Message                   messages         = 3;
-    repeated AccelerometerMeasurement  accelerometers   = 4;
-    repeated BumperMeasurement         bumpers          = 5;
-    repeated CameraMeasurement         cameras          = 6;
-    repeated ForceMeasurement          forces           = 7;
-    repeated Force3DMeasurement        force3ds         = 8;
-    repeated Force6DMeasurement        force6ds         = 9;
-    repeated GyroMeasurement           gyros            = 10;
-    repeated PositionSensorMeasurement position_sensors = 11;
+    uint64                             real_time             = 2;
+    repeated Message                   messages              = 3;
+    repeated AccelerometerMeasurement  accelerometers        = 4;
+    repeated BumperMeasurement         bumpers               = 5;
+    repeated CameraMeasurement         cameras               = 6;
+    repeated ForceMeasurement          forces                = 7;
+    repeated Force3DMeasurement        force3ds              = 8;
+    repeated Force6DMeasurement        force6ds              = 9;
+    repeated GyroMeasurement           gyros                 = 10;
+    repeated PositionSensorMeasurement position_sensors      = 11;
+    OdometryGroundTruth                odometry_ground_truth = 100;
```

Comment:
Why 100? Why not 12?

Reply:
I think it was incase the TC added more stuff into the message

Reply:
Separate it and add a comment to this effect then. Anyone who comes along and doesn't understand why this is the way it is will likely just change it. Also, you should check with the TC to see if they have reserved some IDs, they should use [reserved](https://developers.google.com/protocol-buffers/docs/proto3#reserved) if they want to do that to make sure people don't accidentally use them

Reply:
I've added a comment.